app/helpers/dateTimeUtils.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { isToday } from './dateTimeUtils';
+
+describe('isToday', () => {
```

Comment:
Noticed a bug in `isToday` which is why these specs where added. tbh im not sure how much value these specs in particular add, but it would be nice to encourage more specs to be written in general and this would be a place for folk to build on top of later